using-valtimo/upload/upload-to-documenten-api-with-metadata.md

```diff
@@ -34,3 +34,41 @@ export class AppModule {
   ```
 
 Afterwards, the component is available under the advanced section in the form builder.
+
+## Configuring the Case 'upload process'
+
+When a document is uploaded to a case, the case will search for the connected 'upload process' to handle the upload. The
```

Comment:
`the case will search for the connected 'upload process' to handle the upload.` -> `the 'upload process' connected to the case definition will be started to handle the upload.`